src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.ui;
+
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Scanner;
+
+public class Formatter {
+	
+	/** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+	
+    /**
+     * Empty constructor for Formatter
+     */
+	public Formatter() {
+	}
+	
+	/** Formats a list of strings as a viewable indexed list. */
```

Comment:
Try to make your comment style more consistent. Some of your comments are indented, some are separated into multiple lines.